src/services/updateZendeskTicket.ts

```diff
@@ -1,8 +1,46 @@
-export const updateZendeskTicket = async (
-  requestBody: string | null,
-  message: string
-) => {
-  console.log(
-    `Updating Zendesk Ticket. Request body was ${requestBody}. Message: ${message}`
+import https from 'node:https'
+import { makeHttpsRequest, base64Encode } from './httpsRequestUtils'
+import { retrieveZendeskApiSecrets } from './retrieveZendeskApiSecrets'
+
+export async function updateZendeskTicket(
+  eventBody: string | null,
+  validationMessage: string
+) {
+  if (!eventBody) {
+    console.log('No Zendesk info available. Cannot update ticket.')
+    return
+  }
+  const zendeskTicketInfo = JSON.parse(eventBody)
+  const secrets = await retrieveZendeskApiSecrets()
+  const options: https.RequestOptions = {
+    method: 'PUT',
+    hostname: 'govuk1620731396.zendesk.com',
```

Comment:
I guess we'll get to it, but make sure we put the hostname into an environment variable!